### DISCUSSION TITLE
How to make changes to page data while cloning

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#11874] [Q&A] [answered]
Hi I want to change the data of a page while cloning using the backend code I was able to find the code but unable to make changes to the data can you please help me? please find the below snapshot showcasing the code

![image](https://user-images.githubusercontent.com/25971751/174072107-9d08d98e-5971-46c8-a4ab-594cc8058fe1.png)

**Comment by jptissot** [comment, marked as answer]
You might want to have a look at the `ContentHandler` https://github.com/OrchardCMS/OrchardCore/blob/c8b2954e038aaa54963b60bc6cfe193cfea54799/src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/ContentHandlerBase.cs#L136

https://github.com/OrchardCMS/OrchardCore/blob/0d46fefeab30ed448eb1577341c450ccb9ef92b5/src/OrchardCore.Modules/OrchardCore.Lists/Handlers/ContainedPartHandler.cs#L22
